src/components/PlaceholderContainer/__stories__/PlaceholderContainer.stories.tsx

```diff
@@ -0,0 +1,266 @@
+                <text
+                    fill="#999999"
+                    fontFamily="Sans-serif"
+                    fontSize="26"
+                    strokeWidth="0"
+                    textAnchor="middle"
+                    transform="matrix(1.88041 0 0 1.88041 -48.1289 -81.7475)"
+                    x="86.49"
+                    y="114"
+                >
+                    1:1
+                </text>
+            </g>
+        </svg>
+    );
+};
+
+const contentComponentTest = (
+    <div>
+        <h1>Custom title</h1>
+        <h2>with custom subtitle</h2>
+        <h3>and etc</h3>
+        <p>
+            You can add <strong>here</strong> any long text with custom content and use custom
+            content size for displaying very long texts.
+        </p>
+    </div>
+);
+
+const actionComponentTest = (
+    <div className={b('custom-action')}>
+        <DropdownMenu
+            defaultSwitcherProps={{view: 'flat-secondary'}}
+            items={[
+                {text: 'text 1', action: () => {}},
+                {text: 'text 2', action: () => {}},
+            ]}
+            onSwitcherClick={(e) => e?.stopPropagation()}
+            switcher={
+                <Button>
+                    Text
+                    <Icon data={ChevronDown} size={16} />
+                </Button>
+            }
+        />
+    </div>
+);
+
+const actionMainProps: PlaceholderContainerActionProps = {
+    text: 'Main button',
+    view: 'normal',
+    onClick: () => alert('Click by main button'),
+};
+
+const actionAdditionalBtnProps: PlaceholderContainerActionProps = {
+    text: 'Additional button',
+    view: 'flat-secondary',
+    onClick: () => alert('Click by additional button'),
+};
+
+const baseProps = {
+    title: 'Container with one button & image component',
+    image: <ImageComponentTest />,
+    className: 'placeholder-container',
+};
+
+const placeholderContainerProps: Omit<PlaceholderContainerProps, 'size' | 'direction'> = {
+    ...baseProps,
+    actions: [actionMainProps],
+    align: 'center',
+};
+
+const actionsProps = {
+    actions: [actionMainProps, actionAdditionalBtnProps],
+};
+
+const placeholderContainerCustomRenderedProps: Omit<
+    PlaceholderContainerProps,
+    'size' | 'direction'
+> = {
+    ...placeholderContainerProps,
+    content: contentComponentTest,
+};
+
+const placeholderContainerCustomActionProps: Omit<PlaceholderContainerProps, 'size' | 'direction'> =
+    {
+        ...placeholderContainerProps,
+        actions: actionComponentTest,
+    };
+
+const descriptionProps = {
+    description:
+        'Some long descriptionProps text that can contain of long long very long text etc. It can be repeated like this. Some long descriptionProps text that can contain of long long very long text etc.',
+    promoDescription:
+        "Comparing to 'L' size promo size has full width of the content block, a larger title size and alignment",
+};
+
+export const Size: Story = {
```

Comment:
First story should be named `Default` with the default props, so user can play around with the component